windows/FileUtils.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace CodePush.ReactNative
+{
+    class FileUtils
+    {
+        public async static Task MergeDirectories(StorageFolder source, StorageFolder target)
+        {
+            foreach (StorageFile sourceFile in await source.GetFilesAsync())
+            {
+                await sourceFile.CopyAndReplaceAsync(await target.CreateFileAsync(sourceFile.Name, CreationCollisionOption.OpenIfExists));
```

Comment:
Should we be using [CopyAsync](https://msdn.microsoft.com/en-us/library/windows/apps/br227193.aspx)?